src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/vnet/subnet/_create.py

```diff
@@ -927,9 +933,16 @@ def _build_schema_network_interface_ip_configuration_read(cls, _schema):
             serialized_name="provisioningState",
             flags={"read_only": True},
         )
+        properties.sync_mode = AAZStrType(
+            serialized_name="syncMode",
+        )
```

Comment:
Related to other feature requests, better to move it to another PR to make the current PR more clear.